### DISCUSSION TITLE
Description of the updates

### DISCUSSION THREAD

**Discussion by scriptor-dga** [SkepticMystic/breadcrumbs#251] [Q&A] [answered]
Is there anywhere in this repo a description of all the updates of Breadcrumbs? I saw it was updated today and I'd like to know what has changed. :)

**Comment by SkepticMystic** [comment, marked as answer]
The best place is probably the [Changelog](https://github.com/SkepticMystic/breadcrumbs/blob/master/CHANGELOG.md) :)
Once I get a chance, I add information to the [wiki](https://github.com/SkepticMystic/breadcrumbs/wiki), but that's a slower process

**Comment by michaelpporter** [comment]
Closing as outdated (2022). See the GitHub releases/CHANGELOG for update history.
